src/main/java/com/github/quiram/buildhotspots/visualisation/BuildConfiguration.java

```diff
@@ -63,34 +117,51 @@ private double scalePercentage(byte p_percentage) {
     private double m_origLayoutY = -1;
 
     public int getDepth() {
-        return m_relatedDependencies.stream().map(d -> {
+        return m_Dependencies.stream().map(d -> {
             BuildConfiguration origin = d.getOrigin();
             int depth = 0;
-            if (origin != this) {
-                depth = 1 + origin.getDepth();
-            }
+            depth = 1 + origin.getDepth();
```

Comment:
OK, I think I know what's going on now. m_Dependencies now only has dependencies going up, so the current node will never be origin, so there is no need to check, correct?